wiki/src/markup/view.php

```diff
@@ -513,7 +513,20 @@ function view_replacements($tag, $content)
 
                 return $Stuff;
             break;
-
+            case "text":
+                // I mean, I'm not great at this theming 
+                $content = file_get_contents($content);
+                $content = htmlspecialchars($content);
+                return "<pre style='font:monospace; background-color: #222; padding: 0.5rem;'><code>$content</code></pre>";
+            break;
+            case "code":
+                $content = file_get_contents($content);
+		$content = htmlspecialchars($content);
+                return "<link rel=\"stylesheet\" href=\"//cdnjs.cloudflare.com/ajax/libs/highlight.js/10.5.0/styles/default.min.css\">
+                <script src=\"//cdnjs.cloudflare.com/ajax/libs/highlight.js/10.5.0/highlight.min.js\"></script>
```

Comment:
Ideally we should be loading these dependencies via npm so we can self host them instead of using a 3rd party.

npm dependencies are currently defined here -
https://github.com/wetfish/classic/blob/master/wiki/src/package.json

and there are symlinks to the node_modules folder
https://github.com/wetfish/classic/tree/master/wiki/src/js